### PR TITLE
Adding some schema validation and implementing tests for them

### PR DESCRIPTION
**Pull request by dr-rodriguez**
This is a general enhancement of the schema by adding a few validation functions and establishing tests for them.

### CODE REVIEW

File: simple/schema.py [hunk 1]
```diff
@@ -102,7 +109,7 @@ def validate_ucd(self, key, value):
             raise ValueError(f"UCD {value} not in controlled vocabulary")
         return value
 
-    @validates("effective_wavelength_angstroms")
+    @validates("effective_wavelength")
```
[the remark this answers is not in the document]
Comment (dr-rodriguez) [reply simple/schema.py:112]: Doesn't appear to be renamed in the schema so I got an error when implementing the test. I think it has only changed in the template-db

File: simple/schema.py [hunk 2]
```diff
@@ -169,6 +176,18 @@ class Sources(Base):
...
+        if value > 360 or value < 0:
+            raise ValueError("RA not in allowed range (0..360)")
+        return value
+
+    @validates("dec")
+    def validate_dec(self, key, value):
+        if value > 90 or value < -90:
+            raise ValueError("Dec not in allowed range (-90..90)")
+        return value
+
```
[the remark this answers is not in the document]
Comment (dr-rodriguez) [reply simple/schema.py:190]: Yep, only in the template thus far.

**Comment by kelle** [comment]
We've discussed and I like it now!

**Review by kelle — APPROVED** [review]

**Comment by dr-rodriguez** [comment]
This relates to discussion at https://github.com/astrodbtoolkit/astrodb_utils/issues/43

**Pull request merged**